src/client/components/Navigation/Header.js

```diff
@@ -1,48 +1,69 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
-import { signOut } from '../../firebase/auth';
+import './Header.css';
+import UserIcon from '../../assets/images/icons/user-icon.png'
+// import { signOut } from '../../firebase/auth';
 
-export default function Header({ isAuthenticated = false }) {
-  if (isAuthenticated) {
+export default function Header({ isAuthenticated = false}) {
+  const username = 'Ali Ehaaa';
+
+  if (isAuthenticated) { 
     return (
-      <nav>
-        <ul>
-          <li>
-            <Link to="/">Home</Link>
-          </li>
-          <li>
-            <Link to="/profile">Profile</Link>
-          </li>
-          <li>
-            <button type="button" onClick={signOut}>
-              Sign out
-            </button>
-          </li>
-        </ul>
-      </nav>
-    );
-  }
-  return (
-    <nav>
-      <ul>
-        <li>
-          <Link to="/">Home</Link>
-        </li>
-        <li>
-          <Link to="/sign-in">Sign in</Link>
-        </li>
-        <li>
-          <Link to="/sign-up">Sign up</Link>
+    <div className="header">
+    <div className="home-link">
+    <Link to="/" className="no-design">
+        <span className="review">Review</span><span className="resume">Resume</span>
```

Comment:
In this className review and resume change a color so it will looks like mokeup .To use a write color you can use Figma.

Reply:
The color isn't different, but the boldness.